.travis.yml

```
@@ -1,7 +1,46 @@
+env:
+  global:
+    - VIPS_VERSION_MAJOR=8
+    - VIPS_VERSION_MINOR=7
+    - VIPS_VERSION_MICRO=1
+    - PATH=$HOME/vips/bin:$PATH
+    - LD_LIBRARY_PATH=$HOME/vips/lib:$LD_LIBRARY_PATH
+    - PKG_CONFIG_PATH=$HOME/vips/lib/pkgconfig:$PKG_CONFIG_PATH
+
+dist: trusty
+
+addons:
+  apt:
+    packages:
+      - libexpat1-dev
+      - gettext
+      - liblcms2-dev
+      - libmagickwand-dev
+      - libopenexr-dev
+      - libcfitsio3-dev
+      - libgif-dev
+      - libgs-dev
+      - libgsf-1-dev
+      - libmatio-dev
+      - libopenslide-dev
+      - liborc-0.4-dev
+      - libpango1.0-dev
+      - libpoppler-glib-dev
+      - librsvg2-dev
+      - libwebp-dev
+        # missing on trusty, unfortunately
+        # - libwebpmux2
+      - libfftw3-dev
+      - libglib2.0-dev
+
 language: python
 python:
   - '3.6'
 cache: pip
+before_install:
+  - uname -a
+  - bash install-vips.sh --without-python
```

Can you add a note explaining all of this? Is `uname` just for debugging? Would it work to just do 
```suggestion
  - ./install-vips.sh --without-python
```